src/development/tools/android-studio.md

```diff
@@ -147,7 +147,7 @@ Flutter-specific buttons on the right-hand side of the toolbar.
 
 Flutter offers a best-in-class developer cycle enabling you to see the effect
 of your changes almost instantly with the _Stateful Hot Reload_ feature.
-See [Hot reload](hot-reload) for details.
+See [Hot reload][] for details.
```

Comment:
**chore:** As you touched this line, update for inclusive language and call to action.
```suggestion
To learn more, check out [Hot reload][].
```